src/components/Loader/PersonLink.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+
+export type Props = {
+  person: Person | undefined;
+};
+
+export const PersonLink: FC<Props> = ({ person }) => {
+  if (!person) {
+    return <>-</>;
+  }
```

Comment:
Just add a condition when rendering this component 
`person ? <PersonLink .... `